collections/_pages/smi.html

```diff
@@ -45,13 +45,14 @@
         img.setAttribute('title', result.mesh_name)
 
         switch (result.mesh_name) {
-          case "Open Service Mesh": img.setAttribute('src', 'images/osm.svg'); break;
           case "Traefik Mesh": img.setAttribute('src', 'images/traefik-mesh.svg'); break;
-          case "Istio": img.setAttribute('src', 'images/istio.svg'); break;
+          case "Open Service Mesh": img.setAttribute('src', 'images/osm.svg'); break;
+          case "Nginx": img.setAttribute('src', 'images/nginx-sm.svg'); break;
```

Comment:
Use `Nginx Service Mesh` here.

The names are basically parsed version of the smp spec name (if you want to refer in the future) https://github.com/service-mesh-performance/service-mesh-performance/blob/master/spec/service_mesh.pb.go

Reply:
https://meshery.layer5.io/smi/results/public; it is found out here also mainly; thanks;